exodus/reports/views.py

```diff
@@ -86,32 +86,44 @@ def get_app_icon(request, app_id):
 def get_stats(request):
     from collections import namedtuple
     try:
-        reports = NetworkAnalysis.objects.all()
+        # reports = NetworkAnalysis.objects.all()
         apps = Application.objects.order_by('name', 'handle').distinct('name', 'handle')
-    except:
-        raise Http404("NetworkAnalysis do not exist")
+    except Exception as e:
+        raise Http404(e)
 
     cursor = connection.cursor()
-    cursor.execute("select count(hostname) as score, hostname from reports_dnsquery group by hostname having count(hostname) > 3 order by score desc;")
-    desc = cursor.description
-    nt_result = namedtuple('Result', [col[0] for col in desc])
-    domains = [nt_result(*row) for row in cursor.fetchall()]
-    sum = 0
-    for r in reports:
-        if len(r.dnsquery_set.all()) > 0:
-            sum += 1
-    domain_results = []
-    for d in domains:
-        domain_results.append({'hostname':d.hostname, 'score':int(100.*d.score/sum)})
 
-    cursor.execute("SELECT tt.name, COUNT(*) as c FROM reports_report_found_trackers AS ft, trackers_tracker AS tt WHERE tt.id = ft.tracker_id GROUP BY ft.tracker_id, tt.name ORDER BY c DESC LIMIT 21;")
+    # The part below is unused at the moment (undisplayed stats)
+    domain_results = []
+    # cursor.execute("select count(hostname) as score, hostname from reports_dnsquery group by hostname having count(hostname) > 3 order by score desc;")
+    # desc = cursor.description
+    # nt_result = namedtuple('Result', [col[0] for col in desc])
+    # domains = [nt_result(*row) for row in cursor.fetchall()]
+    # sum = 0
+    # for r in reports:
+    #     if len(r.dnsquery_set.all()) > 0:
+    #         sum += 1
+    # for d in domains:
+    #     domain_results.append({'hostname':d.hostname, 'score':int(100.*d.score/sum)})
```

Comment:
You can delete the commented code. εxodus does not handle network analysis anymore.

Reply:
Alright, I'll clean that up and commit the changes, thanks.

Reply:
Done !